packages/lib/server/getUserFromToken.ts

```diff
@@ -1,13 +1,20 @@
 import { NextApiRequest, NextApiResponse } from "next";
 import prisma from "@documenso/prisma";
+import { authOptions } from "../../../apps/web/pages/api/auth/[...nextauth]";
 import { User as PrismaUser } from "@prisma/client";
 import { getToken } from "next-auth/jwt";
-import { signOut } from "next-auth/react";
+import { getServerSession } from "next-auth/next";
 
 export async function getUserFromToken(
   req: NextApiRequest,
   res: NextApiResponse
 ): Promise<PrismaUser | null> {
+  const oauthUser = await getServerSession(req, res, authOptions);
+
+  if (oauthUser) {
+    return oauthUser.user as PrismaUser;
+  }
```

Comment:
Can we still not use `getToken` here? It should work now with the JWT strategy?

Reply:
Yep. I'll fix it